core/src/main/kotlin/org/evomaster/core/problem/rest/service/SecurityRest.kt

```diff
@@ -0,0 +1,18 @@
+package org.evomaster.core.problem.rest.service
+
+import org.evomaster.core.output.Termination
+import org.evomaster.core.problem.rest.RestIndividual
+import org.evomaster.core.search.Solution
+
+
+/**
+ * Service class used to do security testing after the search phase
+ */
+class SecurityRest {
+
+    fun applySecurityPhase() : Solution<RestIndividual>{
+
```

Comment:
I assume generators for Security tests are going to be written as classes and they will be called in this method. Is that accurate? As example, if I want to test one of the generated endpoints for SQL injection vulnerabilities, where should I implement this test?